integration_tests/integration/bookedtoday/arrivals/matchingRecords/possibleRecordsFound.spec.ts

```diff
@@ -0,0 +1,91 @@
+import ChoosePrisonerPage from '../../../../pages/bookedtoday/choosePrisoner'
+import PossibleRecordsFoundPage from '../../../../pages/bookedtoday/arrivals/matchingRecords/possibleRecordsFound'
+import SearchForExistingPage from '../../../../pages/bookedtoday/arrivals/searchforexisting/searchForExisting'
+import ImprisonmentStatus from '../../../../pages/bookedtoday/arrivals/imprisonmentStatus'
+
+import Page from '../../../../pages/page'
+import Role from '../../../../../server/authentication/role'
+import expectedArrivals from '../../../../mockApis/responses/expectedArrivals'
+
+const matchedRecords = [
+  {
+    firstName: 'Sam',
+    lastName: 'Smith',
+    dateOfBirth: '1970-02-01',
+    prisonNumber: 'G0014GM',
+    pncNumber: '01/1111A',
+    croNumber: '01/0000A',
+    sex: 'MALE',
+  },
+  {
+    firstName: 'Sammy',
+    lastName: 'Smith',
+    dateOfBirth: '1970-02-01',
+    sex: 'MALE',
+  },
+]
+
+context('Possible existing records', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn', Role.PRISON_RECEPTION)
+    cy.task('stubPrison', 'MDI')
+    cy.task('stubAuthUser')
+    cy.task('stubUserCaseLoads')
+    cy.task('stubTransfers', { caseLoadId: 'MDI', transfers: [] })
+    cy.task('stubMissingPrisonerImage')
+    const arrival = expectedArrivals.arrival({
+      fromLocationType: 'COURT',
+      isCurrentPrisoner: false,
+      dateOfBirth: '1972-11-21',
+      pncNumber: null,
+      prisonNumber: null,
+    })
+    cy.task('stubExpectedArrivals', { caseLoadId: 'MDI', arrivals: [arrival] })
+    cy.task('stubExpectedArrival', arrival)
+    cy.task('stubMatchedRecords', matchedRecords)
+    cy.task('stubPrisonerDetails', matchedRecords[0])
+    cy.task('stubImprisonmentStatus')
+    cy.signIn()
+
+    const choosePrisonerPage = ChoosePrisonerPage.goTo()
```

Comment:
these two lines aren't needed as you `goTo` as part of the first test

Reply:
I needed theses to lines in order to set cookie state, which I think is done on selecting a prisoner from the main selection page